### PR TITLE
Remove confirm dialog from Force Full Refresh UI

### PR DESCRIPTION
**Pull request by dannyroberts**
##### SUMMARY
and add a link to docs

Screen Shot 2019-11-13 at 6 07 50 PM
Screen Shot 2019-11-13 at 6 08 05 PM

The link goes to https://confluence.dimagi.com/display/commcarepublic/CommCare+Android+Troubleshooting#CommCareAndroidTroubleshooting-ForceFullRefresh

This is as discussed in https://github.com/dimagi/commcare-hq/pull/25850#issuecomment-553434763

##### PRODUCT DESCRIPTION
Makes the Force Full Refresh UI less cumbersome and scary, linking instead to the docs on the feature.

### CODE REVIEW

File: corehq/apps/users/templates/users/edit_commcare_user.html [hunk 1]
```diff
@@ -70,11 +70,15 @@
           <div class="form form-horizontal">
             <fieldset>
               <legend>{% trans 'Force Full Refresh on next Mobile Sync' %}</legend>
-              <div>
+              <form action="{% url "force_user_412" domain couch_user.user_id %}" method="post">
                 <p>
-                  <a class="btn btn-primary {% if not has_any_sync_logs %} disabled{% endif %}" href="#force_user_412_{{ couch_user.user_id }}" data-toggle="modal" {% if not has_any_sync_logs %}aria-disabled="true"{% endif %}>
+                  See <a href="https://confluence.dimagi.com/display/commcarepublic/CommCare+Android+Troubleshooting#CommCareAndroidTroubleshooting-ForceFullRefresh">{% trans "Force Full Refresh" %}</a> for more information.
```
Comment (orangejenny) [inline corehq/apps/users/templates/users/edit_commcare_user.html]: nitpick: usually HQ opens help links in a new tab

File: corehq/apps/users/templates/users/edit_commcare_user.html [hunk 2]
```diff
@@ -70,11 +70,15 @@
...
+                </p>
+                {% csrf_token %}
+                <p>
+                  <button type="submit" class="btn btn-primary disable-on-submit-no-spinner{% if not has_any_sync_logs %} disabled{% endif %}" {% if not has_any_sync_logs %}disabled{% endif %}>
```
Comment (orangejenny) [inline corehq/apps/users/templates/users/edit_commcare_user.html]: you don't like spinners?

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/users/templates/users/edit_commcare_user.html (hunk 2 above)
Comment (dannyroberts) [reply corehq/apps/users/templates/users/edit_commcare_user.html]: Well, this is using the same icon as the button's icon (meaning "refresh"), so it was weird to then add another one of them next to it that was spinning.

File: corehq/apps/users/templates/users/edit_commcare_user.html (hunk 1 above)
Comment (dannyroberts) [reply corehq/apps/users/templates/users/edit_commcare_user.html]: Wow good eye. Will update

File: corehq/apps/users/templates/users/edit_commcare_user.html (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/users/templates/users/edit_commcare_user.html]: ah yeah...we do that in some places but I don't love it either, and it always seems like a disproportionate amount of work to replace the existing icon

**Pull request merged**